lib/action_client/middleware/response_parser.rb

```diff
@@ -12,7 +12,7 @@ def call(env)
 
         if body.present?
           if content_type.starts_with?("application/json")
-            body = JSON.parse(body)
+            body = JSON.parse(body, object_class: HashWithIndifferentAccess)
```

Comment:
🤯

Reply:
How about `JSON.parse(body, symbolize_keys: true)` instead? I've never seen a case where I wanted a string.

Reply:
I'd be fine with `symbolize_keys`.

I went with `HashWithIndifferentAccess` so I could not change the code in my client project that expected symbols and also not change any code elsewhere in the library that expected strings.

Reply:
According to the docs, the option name is `symbolize_names`: https://ruby-doc.org/stdlib-2.6.3/libdoc/json/rdoc/JSON.html#method-i-parse

Weird!